build actions failing

@claude Run npx --no-install next build
⚠ No build cache found. Please configure build caching for faster rebuilds. Read more: https://nextjs.org/docs/messages/no-cache
Attention: Next.js now collects completely anonymous telemetry regarding usage.
This information is used to shape Next.js' roadmap and prioritize features.
You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
https://nextjs.org/telemetry

  ▲ Next.js 14.2.13

   Creating an optimized production build ...
Error: R] Could not resolve "@/components/ui/checkbox"

    ../../src/components/analyses/prijsherziening/PrijsherzieningDashboard.tsx:6:25:
      6 │ import { Checkbox } from "@/components/ui/checkbox"
        ╵                          ~~~~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@/components/ui/checkbox" as external to exclude it from the bundle, which will remove this error.

Error: R] Could not resolve "@/components/ui/label"

    ../../src/components/analyses/prijsherziening/PrijsherzieningDashboard.tsx:7:22:
      7 │ import { Label } from "@/components/ui/label"
        ╵                       ~~~~~~~~~~~~~~~~~~~~~~~

  You can mark the path "@/components/ui/label" as external to exclude it from the bundle, which will remove this error.

Error: Found 3 problems in 11 documents.

 ├── Couldn't determine the document type for 2 documents. (Skipping documents)
 │   
 │   Please either define a filePathPattern for the given document type definition or provide a valid value for the type field (i.e. the field "type" needs to be one of the following document type names: Analysis).
 │   
 │   • energiekaart-premies/README.md
 │   • energiekaart-premies/update-energiekaart-premies-data.yml
 │   
 └── Encountered unexpected errors while processing of 1 documents. This is possibly a bug in Contentlayer. Please open an issue.
     
     • "prijsherziening-index-i-2021/content.mdx": UnexpectedMDXError: Error: Build failed with 2 errors:
     ../../src/components/analyses/prijsherziening/PrijsherzieningDashboard.tsx:6:25: ERROR: Could not resolve "@/components/ui/checkbox"
     ../../src/components/analyses/prijsherziening/PrijsherzieningDashboard.tsx:7:22: ERROR: Could not resolve "@/components/ui/label"
     

Error: Process completed with exit code 1.

--- a/embuild-analyses/package.json
+++ b/embuild-analyses/package.json
@@ -11,7 +11,9 @@
     "check:faillissementen-geo-join": "node scripts/check-faillissementen-geo-join.js"
   },
   "dependencies": {
+    "@radix-ui/react-checkbox": "^1.1.2",
     "@radix-ui/react-dropdown-menu": "^2.1.16",
+    "@radix-ui/react-label": "^2.1.1",
     "@radix-ui/react-separator": "^1.1.8",
     "@radix-ui/react-slot": "^1.2.4",
     "@radix-ui/react-popover": "^1.1.15",

--- a/embuild-analyses/src/components/ui/checkbox.tsx
+++ b/embuild-analyses/src/components/ui/checkbox.tsx
@@ -1,0 +1,28 @@
+import * as React from "react"
+import * as CheckboxPrimitive from "@radix-ui/react-checkbox"
+import { Check } from "lucide-react"
+
+import { cn } from "@/lib/utils"
+
+const Checkbox = React.forwardRef<
+  React.ElementRef<typeof CheckboxPrimitive.Root>,
+  React.ComponentPropsWithoutRef<typeof CheckboxPrimitive.Root>
+>(({ className, ...props }, ref) => (
+  <CheckboxPrimitive.Root
+    ref={ref}
+    className={cn(
+      "peer h-4 w-4 shrink-0 rounded-sm border border-primary shadow focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-primary data-[state=checked]:text-primary-foreground",
+      className
+    )}
+    {...props}
+  >
+    <CheckboxPrimitive.Indicator
+      className={cn("flex items-center justify-center text-current")}
+    >
+      <Check className="h-4 w-4" />
+    </CheckboxPrimitive.Indicator>
+  </CheckboxPrimitive.Root>
+))
+Checkbox.displayName = CheckboxPrimitive.Root.displayName
+
+export { Checkbox }

--- a/embuild-analyses/src/components/ui/label.tsx
+++ b/embuild-analyses/src/components/ui/label.tsx
@@ -1,0 +1,24 @@
+import * as React from "react"
+import * as LabelPrimitive from "@radix-ui/react-label"
+import { cva, type VariantProps } from "class-variance-authority"
+
+import { cn } from "@/lib/utils"
+
+const labelVariants = cva(
+  "text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+)
+
+const Label = React.forwardRef<
+  React.ElementRef<typeof LabelPrimitive.Root>,
+  React.ComponentPropsWithoutRef<typeof LabelPrimitive.Root> &
+    VariantProps<typeof labelVariants>
+>(({ className, ...props }, ref) => (
+  <LabelPrimitive.Root
+    ref={ref}
+    className={cn(labelVariants(), className)}
+    {...props}
+  />
+))
+Label.displayName = LabelPrimitive.Root.displayName
+
+export { Label }

